pages/tutorials/3.0/system/angle.md

```diff
@@ -0,0 +1,81 @@
+# Handling Angles
+
+## Angles in SFML
+
+SFML defines angles as following the [right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule).
+This means that if the X-axis points to the right in the window and the Y-axis points down, then a positive angle rotates clockwise.
+Encoding these angles in code is handled by [`sf::Angle`](https://www.sfml-dev.org/todo/angle.php "sf::Angle documentation").
+All SFML classes that use angles use this class.
+`sf::Angle` represents an angle in a unit-agnostic manner so you can reason about angles in terms of degrees or radians.
+
+All `sf::Angle` functions are `constexpr`-enabled so anything shown here can be done at compile-time.
+
+## Constructing angles
+
+There are two ways to construct an angle: `sf::degrees` and `sf::radians`.
+
+```cpp
+sf::Angle angle1 = sf::degrees(15);
+sf::Angle angle2 = sf::radians(2);
+```
+
+These two functions construct the angle from either degrees or radians so that you don't have to think about units in your own code.
+Feel free to use either function as you see fit.
+`sf::Angle` will handle all unit conversions internally.
+
+## Manipulating angles
+
+`sf::Angle` provides a suite of operators for arithmetic operations.
+You can add, subtract, multiply, and divide angles.
+There are also modulo, (in)equalty, and comparison operators.
+
+```cpp
+sf::Angle angle1 = sf::degrees(10);
+angle1 *= 2.f; // 20 degrees
+angle1 + sf::radians(0.5f); // 48.6 degrees
+sf::Angle angle2 = -angle1; // -48.6 degrees
+angle2 /= 3.f; // 16.2 degrees
+
+bool equal = (angle1 == angle2); // false
+bool inequal = (angle1 != angle2); // true
+```
+
+`sf::Angle`s can exist outside range [-pi, pi) or [0, 2pi).
+An angle may have a value of 720 degrees, for example.
+If you would like to take an angle and wrap it to its equivalent value within a smaller range, there are two functions for doing so, `sf::Angle::wrapSigned` and `sf::Angle::wrapUnsigned`.
```

Comment:
This paragraph is worded as if wrap methods were mutating in place, which is misleading.